lib/Sema/TypeCheckDecl.cpp

```diff
@@ -1911,21 +1942,20 @@ static void checkGenericParamAccess(TypeChecker &TC,
   if (minAccessScope.isPublic())
     return;
 
-  // Swift 3.0.0 mistakenly didn't diagnose any issues when the context access
-  // scope represented a private or fileprivate level.
-  if (downgradeToWarning == DowngradeToWarning::No) {
-    if (!accessScope.isPublic() &&
-        !isa<ModuleDecl>(accessScope.getDeclContext()) &&
-        TC.getLangOpts().isSwiftVersion3()) {
-      downgradeToWarning = DowngradeToWarning::Yes;
-    }
```

Comment:
What happened to this code?

Reply:
This was a separate patch -- it was duplicated twice so I moved it.

Reply:
…and now it's applied to more places. Maybe it doesn't matter, though.